pyV2DL3/eventdisplay/IrfInterpolator.py

```diff
@@ -68,7 +69,12 @@ def __load_irf(self):
                 )
         self.irf_data = irf_data
         self.irf_axes = irf_axes
-        self.interpolator = RegularGridInterpolator(self.irf_axes, self.irf_data)
+        clk = click.get_current_context()
+        force_extrapolation = clk.params["force_extrapolation"]
```

Comment:
Why do you need to define a variable called `foce_extrapolation`? Couldn't you simply do `if clk.params["force_extrapolation"]` ?